Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/Memory_fwd.h>
+#include <AzCore/Console/ConsoleFunctor.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/string/string_view.h>
+
+namespace AZ
+{
+    class SettingsRegistryInterface;
+}
+
+namespace AzFramework
+{
+    // QualityCVarGroup wraps an integer CVar for a quality group that, when set,
+    // notifies the QualitySystemBus to load all the settings for the group and the
+    // requested level
+    //
+    // Example:
+    // A q_shadows group CVar could be created with 4 levels (low=0,medium=1,high=2,veryhigh=3)
+    // When the level of q_shadows is set to 2 the QualitySystem will be notified
+    // and apply all console settings defined in the SettingsRegistry for that
+    // group at level 2 (high).
+    //
+    class QualityCVarGroup
+    {
+    public:
+        typedef AZ::s64 ValueType;

```

Comment:
`using` is preferred to using `typedef` as the syntax for imore readable when declaring a function pointer.
Also in my opinion makes type assignment(`using Foo = int;`) ;more consistent  with variable assignment (`int foo = 7;`).

Finally `typedef` can't be used to create alias templates while `using` can.
Such as 
```c++
template <class IntType>
using ValueType = IntType;
```

```suggestion
        using ValueType = AZ::s64;

```

Reply:
I decided to use your recommendation of switching to enum class